teradatalabs/mapr52-base/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN yum update -y \
 # CONFIGURE SSH
   && chkconfig sshd on \
   && grep -rl '#Port 22'  /etc/ssh/sshd_config | xargs sed -i 's/#Port 22/Port 22/g' \
+  && service sshd start \
```

Comment:
why would that be needed? This is during image build, the newly started container will have sshd down anyway (except for cases when supervisor.d will spin it up)